src/dev/riscv/LupV.py

```diff
@@ -175,6 +183,11 @@ def generateDeviceTree(self, state):
         cpus_node.append(FdtPropertyWords("timebase-frequency", [10000000]))
         yield cpus_node
 
+        chosen_node = FdtNode("chosen")
+        chosen_node.append(FdtPropertyStrings("stdout-path",
+                            "/soc/lupio-tty@0x20007000"))
```

Comment:
It should technically be "/soc/lupio-tty@20007000" (for some reason, addresses in node names do not contain the `0x` prefix)